From fa44900473cc91308ed44e8a1700f5bd2e1cbf2a Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Thu, 21 Feb 2019 10:11:04 +0100
Subject: [PATCH 1026/1051] phy: core: add PHY_MODE_ETHERNET

Add new PHY's mode to be used by Ethernet PHY interface drivers or
multipurpose PHYs like serdes. It will be reused in further changes.

Change-Id: Id2090d1a239c8469a75a1fb85501fc6a8b0a61a1
Signed-off-by: Grygorii Strashko <grygorii.strashko@ti.com>
Signed-off-by: Kishon Vijay Abraham I <kishon@ti.com>
Reviewed-on: https://sj1git1.cavium.com/4400
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: sa_ip-sw-jenkins
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 include/linux/phy/phy.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index 5407034f8895..97c05ede1e7e 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -43,6 +43,7 @@ enum phy_mode {
 	PHY_MODE_PCIE,
 	PHY_MODE_SATA,
 	PHY_MODE_RXAUI,
+	PHY_MODE_ETHERNET,
 };
 
 /**
-- 
2.17.1

